### DISCUSSION TITLE
sh: decentraland-compiler: command not found

### DISCUSSION THREAD

**Discussion by yaofan-smile** [decentraland/builder#2470] [General]
npm start
> @dcl/builder-site@0.1.0 build:scene
> decentraland-compiler buildScene.json

sh: decentraland-compiler: command not found

**Comment by juanmahidalgo** [comment]
Hi! Did you run the `npm install` command first? Do you see any other errors when running the `npm start` command?

**Reply by yaofan-smile** [reply]
Thank you very much! I forgot to run the command 'npm install'
